assets/css/paybutton-admin.css

```diff
@@ -59,6 +59,13 @@
     border: none;
 }
 
+/* ------------------------------
+   Paywall Settings page styles
+------------------------------ */
+#ecashAddressValidationResult {
```

Comment:
Let's make the address concept generic since we do have support for other blockchains.

`#addressValidationResult` or `#paymentAddressValidationResult` would be good.